api/baseline-api-v1.0.0.yaml

```diff
@@ -0,0 +1,4382 @@
+          description: UUID for the `Workgroup`. _UUID as specified by RFC4122_
+          format: uuid
+          readOnly: true
+          example: bdf94fc0-63ad-4ee9-ac75-7eb63365d0f6
+        created_at:
+          minLength: 1
+          type: string
+          description: Date and time of record creation. Expressed as a Unix timestamp.
+          format: date-time
+          readOnly: true
+          example: '2021-05-21T12:36:52.977Z'
+        network_id:
+          type: string
+          description: UUID for the host `Network` of the `Workgroup`. _UUID as specified by RFC4122_
+          format: uuid
+          example: 07102258-5e49-480e-86af-6d0c3260827d
+        user_id:
+          maxLength: 36
+          minLength: 1
+          type: string
+          description: UUID for the `User` that created the `Workgroup`. _UUID as specifed by RFC4122_
+          format: uuid
+          example: 7c8fe6f1-38c3-4da1-b4b7-7591c6d0ca7c
+        name:
+          minLength: 1
+          type: string
+          description: User-defined name of the `Workgroup`
+        description:
+          type: string
+          description: User-defined `Workgroup` descriptor
+          example: Sample workgroup description
+        type:
+          minLength: 1
+          type: string
+          description: Designates `Workgroup` `type`. In a baseline `Workgroup`, will always be `baseline`
+          example: baseline
+          default: baseline
+        config:
+          $ref: '#/components/schemas/WorkgroupConfig'
+        hidden:
+          type: boolean
+          description: When `True`, the `Workgroup` is hidden from public searches
+      description: |-
+        A `Workgroup` represents a logical collection of resourcess provisioned within the baseline containerized microservices architecture. An `Workgroup` is immutably configured upon initialization with support for a primary 'Network'.
+        Access to programmatically manage resources associated with an `Workgroup` is provided via REST API, to which ACL-based authorization is managed using generic permissions.
+      example:
+        Demo Workgroup:
+          id: bdf94fc0-63ad-4ee9-ac75-7eb63365d0f6
+          created_at: '2021-05-21T12:36:52.977Z'
+          network_id: 07102258-5e49-480e-86af-6d0c3260827d
+          user_id: 7c8fe6f1-38c3-4da1-b4b7-7591c6d0ca7c
+          name: Demo Workgroup
+          description: Workgroup intended for demonstration purposes
+          type: baseline
+          config:
+            baselined: true
+            webhook_secret: 1dfd34519a06420d8a6ed18afdffe932
+          hidden: false
+    WorkgroupConfig:
+      title: Workgroup Config
+      type: object
+      properties:
+        baselined:
+          type: boolean
+        webhook_secret:
+          minLength: 1
+          type: string
+        network_id:
+          type: string
+      description: Configuration object intended for use by integrated services
+    Workflow:
```

Comment:
see PR #56 for a required definition of workflow.

Reply:
addressed--